mesh/amqp.c

```diff
@@ -430,6 +440,84 @@ static void connect_with_delay(struct amqp_thread_context *context, int delay)
 	set_amqp_state(context, AMQP_STATE_CONNECTING);
 }
 
+static bool amqp_consume(struct amqp_thread_context *context)
+{
+	amqp_rpc_reply_t reply;
+
+	// Who should do this Gateway app / BlueZ?
+	char *exchange = l_strdup_printf("rc.%s", strchr(context->config.exchange, '.') + 1);
```

Comment:
Maybe set `config.exchange` to just the project id and prepend ether `mon` or `rc` on bluez side?